app/controllers/tag_set_nominations_controller.rb

```diff
@@ -312,10 +312,10 @@ def collect_update_multiple_results
             # this is the tricky one: make sure we can do this name change
             tagnom = TagNomination.for_tag_set(@tag_set).where(type: "#{type.classify}Nomination", tagname: name).first
             if !tagnom
-              @errors << ts("Couldn't find a #{type} nomination for #{name}")
+              @errors << ts("Couldn't find a #{type} nomination for %{name}", name: name)
```

Comment:
I changed the handling of `name` but not `type` here because `type` is the tag type, which should be translated once this message is moved over to `t()`.